cEP-0021.md

```diff
@@ -0,0 +1,134 @@
+# Profile Bears
+
+| Metadata |                                                   |
+| -------- | ------------------------------------------------- |
+| cEP      | 0021                                              |
+| Version  | 1.0                                               |
+| Title    | Debug and Profile Bears                           |
+| Authors  | Vaibhav Kumar Rai <mailto:raivaibhav08@gmail.com> |
+| Status   | Proposed                                          |
+| Type     | Process                                           |
+
+## Abstract
+
+This cEP describe the detailed implementation of profiling interface as a part
+of the
+[GSoC'18 project](https://summerofcode.withgoogle.com/projects/#6109762077327360).
+
+## Introduction
+
+Profiling a Python program is form of dynamic analysis that measures the
+execution time of the program and the functions it is composed of. This will
+give the data about where our program is spending time, and what area might
+be worth optimizing. Thus we can provide a *Profiling interface* from within
+coala and make it available to bear writers so that they can create Better
+Performant bears.
+
+This cEP proposes to introduce the Profiling interface so that bear writers
+will have the ability to Profile the Bear's code to to optimize it's
+performance.
+
+## Proposed Change
+
+Bear writers can invoke the profiler with the `--profile-bears`,
```

Comment:
with the `--profile-bears` argument.

last word is missing. Also period instead of comma.